python_bindings/README.md

```diff
@@ -0,0 +1,33 @@
+# A python wrapper for easi
+
+Install with `-DPYTHON_BINDINGS=ON` cmake option.
+
+Add the library to your path:
+
+```bash
+export PYTHONPATH=your_build_folder>python_bindings:$PYTHONPATH
+```
+
+The module can then be used as follows:
+
+```python
+import numpy as np
+import easi
+
+coords = np.random.rand(10,3)
+region = np.random.randint(1,3, 10)
+print(easi.evaluate_model(coords, region, ["rho", "lambda"], "test.yaml"))
```

Comment:
When I try this, I get this error:
```
Traceback (most recent call last):
  File "/opt/easi/pytest.py", line 2, in <module>
    import easi
ModuleNotFoundError: No module named 'easi'
```

Reply:
how did you install it? with spack or manually?
If you install it manually, you need to do:
```export PYTHONPATH=~/SeisSol/easi/build/python_bindings:$PYTHONPATH``` or similar.
Note that the version of python (e.g. 3.10 `easi.cpython-310-x86_64-linux-gnu.so`) has to be the same as the python version that you use for loading the module.
(this may happen if the python in spack ~/.spack/packages.yaml is the system python, and your python is a conda installed python).